.github/workflows/continuous-integration.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - name: Build libraries and tests
         run: |
           cmake -S . -B build -G "Ninja" \
-              -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+              -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Building Fortran code with `-O3` is a slippery slope in my experience. Alas, I can't tell more than that at the moment.